	# Get this puppy working with kernel 2.5.x
	# <azarah@gentoo.org> (06 March 2003)
--- eject.c.orig
+++ eject.c
@@ -58,7 +58,13 @@
 #include <sys/ioctl.h>
 #include <sys/wait.h>
 #include <sys/mtio.h>
-#include <linux/types.h>
+#ifndef __KERNEL__
+# define __KERNEL__
+# include <linux/types.h>
+# undef __KERNEL__
+#else
+# include <linux/types.h>
+#endif
 #include <linux/cdrom.h>
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2,1,0)
 #include <linux/ucdrom.h>
